readme.txt

```diff
@@ -22,6 +22,9 @@ This section describes how to install the plugin and get it working.
 
 == Changelog ==
 
+= 2.1.10 =
```

Comment:
I think we can't preemptively add the next version since it's not decided if it's going to be a major/minor/patch version. Usually, I'd set it to `= Unreleased =` or just leave it empty

Reply:
Thanks! FIxed in [ffb9afd](https://github.com/Automattic/wc-calypso-bridge/pull/1203/commits/ffb9afd6a1bc82fcd3bf80c67b7981092ef5eed6)